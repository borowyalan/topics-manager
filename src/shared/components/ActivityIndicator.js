import React from 'react';

export const ActivityIndicator = () => {
    return (
        <p>Loading</p>
    );
};
