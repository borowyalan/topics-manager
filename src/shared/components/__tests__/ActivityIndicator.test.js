import React from 'react';
import { configure, shallow } from 'enzyme';
import { ActivityIndicator } from '../ActivityIndicator';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('<ActivityIndicator>', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<ActivityIndicator/>);
    });
    
    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
