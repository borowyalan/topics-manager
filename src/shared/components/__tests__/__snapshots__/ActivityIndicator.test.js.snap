// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActivityIndicator> should match snapshot 1`] = `
<p>
  Loading
</p>
`;
